Error on GitHub Actions

GitHub Actionsで `npm install @vivliostyle/cli --save` ViblioStyleが実行できません。

```
Run npx vivliostyle -v
npx vivliostyle -v
  shell: /bin/bash -e {0}
cli: 4.2.0
core: 2.10.0

```

```
  npx vivliostyle build -t 240 publish/buildHtml.html -o publish/refs/tags/galley-debug-version-of-vivliostyle.pdf
  shell: /bin/bash -e {0}
- Collecting build config
ℹ Rendering browser (Chromium) is not installed yet. Downloading now...
- Downloading Browser
✔ Successfully downloaded browser
- Downloading Browser: [=============== ] 111.4 Mb / 116.3 Mb
Error: Evaluation failed: TypeError: Cannot read properties of null (reading 'getMetadata')

    at t.n.getMetadata (file:///Users/runner/work/[project-path]/node_modules/@vivliostyle/viewer/lib/js/vivliostyle-viewer.js:14:694652)
If you think this is a bug, please report at https://github.com/vivliostyle/vivliostyle-cli/issues
    at puppeteer_valuation_script__:1:26
    at ExecutionContext._evaluateInternal (/Users/runner/work/[project-path]/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:221:19)
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async ExecutionContext.evaluate (/Users/runner/work/[project-path]/node_modules/puppeteer-core/lib/cjs/puppeteer/common/ExecutionContext.js:110:16)
    at async Object.buildPDF (/Users/runner/work/[project-path]/node_modules/@vivliostyle/cli/dist/pdf.js:160:22)
    at async build (/Users/runner/work/[project-path]/node_modules/@vivliostyle/cli/dist/commands/build.js:93:26)

Error: Process completed with exit code 1.
```

--- a/src/pdf.ts
+++ b/src/pdf.ts
@@ -220,9 +220,6 @@ export async function buildPDF({
   await page.goto(navigateURL, { waitUntil: 'networkidle0' });
   await page.waitForFunction(() => !!window.coreViewer);
 
-  const metadata = await loadMetadata(page);
-  const toc = await loadTOC(page);
-
   await page.emulateMediaType('print');
   await page.waitForFunction(
     () => window.coreViewer.readyState === 'complete',
@@ -235,6 +232,10 @@ export async function buildPDF({
   if (lastEntry) {
     logSuccess(stringifyEntry(lastEntry));
   }
+
+  const metadata = await loadMetadata(page);
+  const toc = await loadTOC(page);
+
   startLogging('Building PDF');
 
   const pdf = await page.pdf({

